physical/raft/raft.go

```diff
@@ -1933,6 +2017,186 @@ func (l *RaftLock) Value() (bool, string, error) {
 	return true, value, nil
 }
 
+func fileExists(name string) (bool, error) {
+	_, err := os.Stat(name)
+	if err == nil {
+		// File exists!
+		return true, nil
+	}
+	if errors.Is(err, os.ErrNotExist) {
+		return false, nil
+	}
+	// We hit some other error trying to stat the file which leaves us in an
+	// unknown state so we can't proceed.
+	return false, err
+}
+
+func parseRaftBackendConfig(conf map[string]string, logger log.Logger) (*RaftBackendConfig, error) {
+	c := &RaftBackendConfig{}
+
+	c.Path = conf["path"]
+	envPath := os.Getenv(EnvVaultRaftPath)
+	if envPath != "" {
+		c.Path = envPath
+	}
```

Comment:
This flips the logic from what was there before so at first I thought we changed the precedence of config vs env. But I see this is equivalent but just simplified.